basket/news/tasks.py

```diff
@@ -1113,10 +1138,20 @@ def process_petition_signature(data):
     user_data = get_user_data(email=data["email"], extra_fields=["id"])
     if user_data:
         sfdc.update(user_data, contact_data)
+        ctms_data = contact_data.copy()
+        del ctms_data["_set_subscriber"]
+        ctms.update(user_data, ctms_data)
     else:
         contact_data["token"] = generate_token()
         contact_data["email"] = data["email"]
+        ctms_data = contact_data.copy()
         contact_data["record_type"] = settings.DONATE_CONTACT_RECORD_TYPE
+
+        del ctms_data["_set_subscriber"]
+        contact = ctms.add(ctms_data)
```

Comment:
Here's another spot where we could be setting ``mofo_relevant``.

Reply:
My opinion is that anything coming from the MoFo queue should get `mofo_relevant`. But keep in mind that MoFo plans to turn off this queue in favor of Cinchy handling the queue.